x-pack/plugins/fleet/common/services/validate_package_policy.ts

```diff
@@ -217,7 +217,7 @@ export const validatePackagePolicyConfig = (
   }
 
   if (varDef.required) {
-    if (parsedValue === undefined || (typeof parsedValue === 'string' && !parsedValue)) {
+    if (parsedValue === undefined || (varDef.type === 'yaml' && !parsedValue)) {
```

Comment:
Should we make the YAML case more explicit and not rely on empty string falsey-ness?
```suggestion
    if (parsedValue === undefined || (varDef.type === 'yaml' && parsedValue === '')) {
```